### PR TITLE
Rating grid 

### PR DESCRIPTION
**Pull request by AagmanSuri**
In this branch their are two components ratings and grid .

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                        <div class="child">5</div>
+                        <div class="child">6</div>
+                    </div>
+
+                </div>
+
+
+
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/NJKkJM9eaeXAenJsN7TI"
+                    style="width: 403px; height: 353px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dhimanaishwarya) [inline Components/Grid/grid.html:145]: suggestion: you can use a class to style iframe to maintain uniformity

**Review by dhimanaishwarya — APPROVED** [review]
your code looks clean and good to merge

File: Components/Grid/grid.html (hunk 1 above)
Comment (AagmanSuri) [reply Components/Grid/grid.html:145]: Thanks for the review [dhimanaishwarya](https://github.com/dhimanaishwarya)

**Comment by AagmanSuri** [comment]
> looks good to go 👍

Thanks for the review [Ananyamadhu08](https://github.com/Ananyamadhu08)

**Pull request merged**